### DISCUSSION TITLE
请问能否支持关闭窗口时保留托盘区？

### DISCUSSION THREAD

**Discussion by Zonz-Ly** [tw93/Pake#585] [Q&A] [answered]
请问能否让打包的 .exe 支持点关闭按钮后隐藏于系统托盘区而不是直接退出，点系统托盘区图标重新唤出窗口，同时保留原有网页进度？

- #517 是一个类似的问题，但 Windows 下 Ctrl+W 是无效的。
- 托盘区点右键唤出的菜单是有效的，但点击托盘区图标是无效的，即隐藏时无法便捷唤出窗口
![屏幕截图_20231009181021](https://github.com/tw93/Pake/assets/32699823/5a43ed3e-f9b7-4ca0-b409-15f577b0813c)
- macOS 下绝大多数软件和 Windows 下的 QQ、Wechat、网易云音乐等软件的调教让我习惯直接点关闭按钮，但是这样软件就直接关掉了，重新打开进度就丢失了，很烦人。

**Comment by Zonz-Ly** [comment, marked as answer]
找到解决方案了，自己写了个 AHK。
[关闭按钮或Ctrl+W隐藏窗口而不关闭.zip](https://github.com/tw93/Pake/files/13069800/Ctrl%2BW.zip)
使用方法：
- 解压后，将 .exe 文件重命名为与目标软件相同的文件名，打开即可；如：需要对「NotePad.exe」起作用，则将文件命名为「NotePad.exe」。理论上适用于任何软件。
- 点击关闭按钮或按下 Ctrl+W 组合键，窗口将被隐藏，可以从**任务栏**中重新打开；
- 按下 Ctrl+Q 组合键，窗口将被关闭。
- 如需开机启动，请按下 Win+R 组合键，输入「shell:startup」，并按下回车键，将上述改名后的文件放在弹出的文件夹中。
- 如需对多款软件起作用，则多复制几份，一款软件一份，分别按上述规则分别重命名即可。
- 如需退出这款小工具，请在系统托盘区找到图标并点击右键，选择「Exit」。

**Comment by 1i9htning** [comment]
要是Pake打包好的应用程序自身就支持就好了
